wgpu-core/src/command/draw.rs

```diff
@@ -245,3 +250,115 @@ pub enum RenderCommand {
     EndPipelineStatisticsQuery,
     ExecuteBundle(id::RenderBundleId),
 }
+
+/// Equivalent to `RenderCommand` with the Ids resolved into resource Arcs.
+#[doc(hidden)]
+#[derive(Clone, Debug)]
+pub enum ArcRenderCommand<A: HalApi> {
```

Comment:
As long as the long term goal of this is to remove the duplication, I'm fine with this for now.